webapps/src/main/webapp/javascript/automatic-translation/extensions.js

```diff
@@ -100,6 +100,17 @@
       },
     });
 
+    extensionRegistry.registerExtension('automatic-translation', 'action', {
+      id: 'auto-translate',
+      rank: 1000,
+      isEnabled: () => true,
+      labelKey: 'UIActivity.label.translate',
+      click: (message, callback) => {
+        fetchTranslation(message).then(translated => {
+          callback(translated.translation);
+        });
+      },
+    });
```

Comment:
why we add this extension ?

Reply:
> why we add this extension ?

for the translate button in notes and news ?

Reply:
Ok, the name of the extension is disturbing. 
You named it 'automatic-translation', but should it be something like 'notes-translation' and 'news-translation' ?
For activities, the xtension is named "ActivityContent" or ('activity', 'comment-action').

Reply:
WDYT ?

Reply:
Ok, but i wanted to have one extension for both. that's why. 
And i really don't know which is better: define one extension or two extensions(news-extension and notes-extension).

Reply:
By using only one extension point, you will not be able in the future to add something in the news extension point but not in the notes extension point.
For me, it is better to have 2 extensions point, with a different name